dation.css

```diff
@@ -0,0 +1,703 @@
+    margin: var(--margin-m);
+}
+
+.card-modern {
+    border-radius: var(--border-radius-s);
+}
+
+.card-classic {
+    border: 1px solid black;
+}
+
+.card-horizontal {
+    display: inline-block;
+    position: relative;
+    height: var(--card-width);
+    width: var(--card-height);
+    margin: var(--margin-m);
+}
+
+.card-text {
+  background: linear-gradient(to right bottom, white, #10B981);
+}
+
+.card-img {
+    width: 100%;
+    height: auto;
+    max-width: 100%;
+}
+
+.card-content {
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    background-color: var(--primary-color);
+    color: var(--secondary-color);
+    width: 90%;
+    bottom: 5%;
+    text-align: center;
+    left: 0;
+    right: 0;
+    margin-left: auto;
+    margin-right: auto;
+    padding: var(--padding-xs);
+}
+
+.card-dismiss {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: absolute;
+    width: 30px;
+    height: 30px;
+    top: 3%;
+    right: 3%;
+    border-radius: 50%;
+    background-color: var(--btn-txt-color);
+    color: var(--btn-bg-color);
+    cursor: pointer;
+}
+
+.list li {
+    list-style: none;
+    margin: var(--margin-s);
+    padding: var(--padding-s);
+    text-decoration: none;
+    color: black;
+}
+
+li a {
+    text-decoration: none;
+    color: black;
+}
+
+.nav {
+    width: 100%;
+    position: sticky;
+}
+
+.nav-classic {
+    display: flex;
+    background-color: var(--nav-bg-color);
+    color: var(--nav-txt-color);
+    justify-content: space-between;
+    align-items: center;
+}
+
+
+.site-nav-left {
+    display: flex;
+    overflow-x: auto;
+    overflow-y: hidden;  
+}
+
+.site-nav-left li {
+    list-style: none;
+    padding: var(--padding-s);
+    width: auto;
+}
+
+.site-nav-left li a, .nav-modern a {
+    text-decoration: none;
+    color: var(--nav-txt-color);
+}
+
+.site-nav-right {
+    display: flex;
+    margin: 0 var(--margin-m);
+}
+
+.nav-modern {
+    position: fixed;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    top: 10;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    z-index: 2;
+    width: 100vw;
+    height: 0;
+    background-color: var(--primary-color);
+    color: var(--secondary-color);
+    transition: 0.5s;
+}
+
+.nav-modern ul {
+    list-style: none;
+    height: 40%;
+    width: 100%;
+    text-align: center;
+    overflow-y: auto;
+}
+
+.nav-modern ul li {
+    text-decoration: none;
+    font-size: 2em;
+    margin: var(--margin-s) auto;
+}
+
+.top {
+    background-color: var(--primary-color);
+    color: var(--secondary-color);
+    top:-5px;
+    right: -5px;
+}
+
+.bottom {
+    bottom: 7%;
+    right: 10%;
+    border: 2px solid white;
+}
+
+.online{
+    background-color: green;
+}
+
+.offline {
+    background-color: red;
+}
+
+.shadow {
+    box-shadow: 0.3em 0.5em 1em var(--shadow-color);
+}
+
+.glow {
+    box-shadow: 0 0 1em var(--glow-color);
+}
```

Comment:
glow shadow on avatar looks a bit odd. try changing the shadow color, might look good

Reply:
Changed it to magenta. Looks good to me. It is any way a variable so can be over ridden easily so that's not a problem.